src/config/config.go

```diff
@@ -42,3 +44,11 @@ func Parse(config []byte) (cfg Config, err error) {
 	}
 	return
 }
+
+func (cfg Config) WriteFile(path string, mode os.FileMode) (err error) {
```

Comment:
I only did that because I was using config.Parse to load the config, it seemed an abstraction violation to assume writing json to the cache would be appropriate for config.Parse to consume.

What I'll do instead is just json.Marshal/json.Unmarshal the cached config in acquireConfig() and remove the config.WriteFile().